panels/user-accounts/data/account-dialog.ui

```diff
@@ -277,6 +277,22 @@
                       </packing>
                     </child>
                     <child>
+                      <object class="GtkRadioButton" id="local_no_password_radio">
+                        <property name="label" translatable="yes">Allow user to login _without a password</property>
```

Comment:
Thought so too but there's already:
_Allow user to set a password when they next login_